qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css

```diff
@@ -121,3 +121,15 @@ footer .helpful-container .was-helpful-thank-you {
   background-color: var(--qiskit-color-purple);
   color: #ffffff;
 }
+
+/* ---------------------------------------------------------------------------
+* Tutorials
+* ---------------------------------------------------------------------------- */
+
+/* Don't have a purple border. This rule overrides `.npshinx-gallery > a`.
+*
+* We keep the default style of the text itself being purple to make it more clear that this is a
+* clickable link (better accessibility).*/
```

Comment:
hmm I think it would be better to keep the underline the same colour as it looks like that is what we do for the rest of qiskit.org, although we only show the underlines on hover. I'm not sure how furo handles it, is the underline always there or only on hover?
<img width="572" alt="Screenshot 2023-06-15 at 9 50 24 AM" src="https://github.com/Qiskit/qiskit_sphinx_theme/assets/23662430/83dc741f-963b-4bf2-b0dd-4d4bf643168c">
<img width="244" alt="Screenshot 2023-06-15 at 9 49 23 AM" src="https://github.com/Qiskit/qiskit_sphinx_theme/assets/23662430/894c1e2a-076e-4e78-a99f-45b41aac7756">
<img width="499" alt="Screenshot 2023-06-15 at 9 49 14 AM" src="https://github.com/Qiskit/qiskit_sphinx_theme/assets/23662430/541e548d-c9aa-40c9-ab84-bac4d8dc0746">

Reply:
I think the ither qiskit.org example you screenshotted are different bc they are CTAs and have arrow icons, so the underline isn't as necessary to imply that it is a link

Reply:
Okay. I'd like to discuss in the docs squad meeting on Tuesday how we should handle links then. This discussion impacts far more than this tutorial box—it's every link on the site. I opened https://github.com/Qiskit/qiskit_sphinx_theme/issues/385 to track this.

@javabster could we please merge this PR in the meantime? It's still some forward progress.